src/main/java/stevesaddons/asm/StevesHooks.java

```diff
@@ -29,28 +32,35 @@
 import stevesaddons.api.IHiddenTank;
 import stevesaddons.components.ComponentMenuTriggered;
 import stevesaddons.helpers.StevesEnum;
+import stevesaddons.interfaces.GuiRFManager;
 import stevesaddons.naming.BlockCoord;
 import stevesaddons.naming.NameRegistry;
 import stevesaddons.reference.Null;
 import stevesaddons.threading.SearchItems;
 import vswe.stevesfactory.Localization;
 import vswe.stevesfactory.blocks.ConnectionBlock;
+import vswe.stevesfactory.blocks.TileEntityCluster;
 import vswe.stevesfactory.blocks.TileEntityManager;
+import vswe.stevesfactory.blocks.TileEntityRFCluster;
+import vswe.stevesfactory.compat.Hooks;
+import vswe.stevesfactory.components.CommandExecutorRF;
 import vswe.stevesfactory.components.ComponentHelper;
 import vswe.stevesfactory.components.ComponentMenuItem;
 import vswe.stevesfactory.components.ComponentType;
 import vswe.stevesfactory.components.Connection;
+import vswe.stevesfactory.components.ConnectionOption;
+import vswe.stevesfactory.components.ConnectionSet;
 import vswe.stevesfactory.components.FlowComponent;
 import vswe.stevesfactory.components.ScrollController;
 import vswe.stevesfactory.network.DataReader;
 import vswe.stevesfactory.network.DataWriter;
 import vswe.stevesfactory.settings.Settings;
 
-public class StevesHooks {
+public class StevesHooks implements Hooks {
 
     public static final Multimap<TileEntityManager, FlowComponent> delayedRegistry = HashMultimap.create();
 
-    public static void addCopyButton(final TileEntityManager manager) {
+    public void addCopyButton(final TileEntityManager manager) {
```

Comment:
Please, annotate methods with `@Override`.
Also (since there's no ASM) this class can be moved to another package.

Reply:
I tried before, but that broke the nightly and I have no idea how to debug it - it didn't throw an error, it just froze at some point.